packages/playwright/src/common/process.ts

```diff
@@ -54,12 +53,7 @@ process.on('disconnect', () => gracefullyCloseAndExit(true));
 process.on('SIGINT', () => {});
 process.on('SIGTERM', () => {});
 
-// Clear execArgv immediately, so that the user-code does not inherit our loader.
-process.execArgv = execArgvWithoutExperimentalLoaderOptions();
-
-// Node.js >= 20
```

Comment:
Was it actually >= 18 or we still need the check?

Reply:
It was >= 18, but we didn't know that at the time of writing the comment.